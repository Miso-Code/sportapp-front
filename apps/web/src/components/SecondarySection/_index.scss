.section-secondary {
	@apply w-1/2 lg:block hidden;

	.container-image {
		@apply w-full h-full m-0;
	}

	.image-register {
		@apply object-cover object-center w-full h-full;
	}
}
