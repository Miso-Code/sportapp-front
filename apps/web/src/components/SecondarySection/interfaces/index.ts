export interface Props {
	image: string
	altImage: string
}
