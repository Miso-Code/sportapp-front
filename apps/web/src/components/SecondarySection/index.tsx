import 'components/SecondarySection/_index.scss'
import { Props } from './interfaces'

export default function SecondarySection({ image, altImage }: Props) {
	return (
		<section className='section-secondary'>
			<figure className='container-image'>
				<img className='image-register' src={image} alt={altImage} />
			</figure>
		</section>
	)
}
