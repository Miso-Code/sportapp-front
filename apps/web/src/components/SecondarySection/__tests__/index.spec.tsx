import SecondarySection from 'components/SecondarySection'
import { RenderResult, render } from '@testing-library/react'

describe('SecondarySection', () => {
	let wrapper: RenderResult

	beforeEach(() => {
		wrapper = render(<SecondarySection image='image' altImage='altImage' />)
	})

	afterEach(() => {
		wrapper.unmount()
	})

	it('should render correctly', () => {
		expect(wrapper.container).toMatchSnapshot()
	})
})
