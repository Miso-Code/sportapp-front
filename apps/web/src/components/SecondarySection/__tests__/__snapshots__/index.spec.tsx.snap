// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondarySection should render correctly 1`] = `
<div>
  <section
    class="section-secondary"
  >
    <figure
      class="container-image"
    >
      <img
        alt="altImage"
        class="image-register"
        src="image"
      />
    </figure>
  </section>
</div>
`;
